src/screens/Customers/CustomerMain.tsx

```diff
@@ -1,228 +1,243 @@
       right: theme.spacing(2),
       color: 'white',
     },
-  });
-
-interface CustomerMainProps extends RouteComponentProps {
-  classes: { title: string; headerWrapper: string; selectionHeader: string; rightAlign: string; fab: string; };
-  customers: CustomerRecord[]
-}
-
-enum SortBy {
-  NAME = 'Name (A - Z)' as any,
-  METER = 'Meter Number' as any
-}
+    searchIcon: {
+      color: theme.palette.text.secondary,
+      marginRight: '40px',
+    },
+    searchBar: {
+      zIndex: 1,
+      position: 'absolute',
+      marginTop: '-20px',
+      width: '90%',
+      height: '20px',
+      backgroundColor: 'white',
+    },
+    tab: {
+      color: `${theme.palette.divider}`,
+      fontSize: '12px',
+      '&:focus': {
+        outline: 'none',
+      },
+    },
+    active: {
+      height: '7px',
+      width: '7px',
+      borderRadius: '50%',
+      backgroundColor: theme.palette.secondary.light,
+      marginRight: '5px',
+    },
+    activeContainer: {
+      padding: '10px 5px',
+      display: 'inline-flex',
+      alignItems: 'center',
+    },
+    tabIcon: {
+      fontSize: '15px',
+    },
+    tabLabel: {
+      display: 'inline-flex',
+    },
+    tabContent: {
+      marginBottom: '50px',
+    },
+  }));
 
-enum FilterBy {
-  PAYMENT_STATUS = 'Payment Status' as any,
-  METER_STATUS = 'Meter Status' as any,
-  ACTIVE_STATUS = 'Active Status' as any,
+interface CustomerMenu {
+  all: CustomerRecord[];
+  toMeter: CustomerRecord[];
+  toCollect: CustomerRecord[];
+  done: CustomerRecord[];
 }
 
-type FilterByLabel = Record<keyof typeof FilterBy, string[]>;
-
-const labels: FilterByLabel = {
-  'PAYMENT_STATUS': ['Unpaid', 'Paid'],
-  'METER_STATUS': ['Has Meter', 'No Meter'],
-  'ACTIVE_STATUS': ['Active', 'Inactive']
+interface CustomerMainProps extends RouteComponentProps {
+  customers: CustomerRecord[];
+  match: any;
 }
 
 function CustomerMain(props: CustomerMainProps) {
-  const { classes } = props;
-  const [filteredCustomers, setFilteredCustomers] = useState<CustomerRecord[]>([]);
-  const [filteredCustomersAlt, setFilteredCustomersAlt] = useState<CustomerRecord[]>([]);
-  const [allCustomersTrie, setAllCustomersTrie] = useState<TrieTree<CustomerRecord>>(new TrieTree('name'));
-  const [sortBy, setSortBy] = useState<SortBy>(SortBy.NAME);
-  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.ACTIVE_STATUS);
-  const [sortAndFilter, setSortAndFilter] = useState<string[]>([])
-  const [filterLabels, setFilterLabels] = useState<string[]>(labels["ACTIVE_STATUS"]);
+  const classes = styles(props);
+  const { match } = props;
   const [searchValue, setSearchValue] = useState<string>("");
-  const fullCustomers: CustomerRecord[] = useSelector(selectAllCustomersArray) || [];
+  const [tabValue, setTabValue] = useState<CustomerStatus>(CustomerStatus.ALL);
+  const changeTab = (event: React.ChangeEvent<{}>, newValue: CustomerStatus) => {
+    setTabValue(newValue);
+  };
+
+  //default customer arrays
```

Comment:
Nit: could you add spaces after the `//` and capitalize just to make it look cleaner?